[bn]  Localize virtualization.md

PrashantDesale2004 opened the pull request:
#2403
- corrected Grammatical Errors in Bengali Version of Virtualization

sajibAdhi commented:
Hey @PrashantDesale2004
Kindly set the targeted branch from `main` to `dev-bn`. You can use `[bn] localization update content/bn/virtualization.md` as PR as I have changed it.

PrashantDesale2004 commented:
> Hey @PrashantDesale2004 Kindly set the targeted branch from `main` to `dev-bn`. You can use `[bn] localization update content/bn/virtualization.md` as PR as I have changed it.

i didn't get it. Do you mean i have to specify "[bn] localization update content/bn/virtualization.md"  in commit meassag. can you elaborate??

sajibAdhi commented:
> > Hey @PrashantDesale2004 Kindly set the targeted branch from `main` to `dev-bn`. You can use `[bn] localization update content/bn/virtualization.md` as PR as I have changed it.
> 
> i didn't get it. Do you mean i have to specify "[bn] localization update content/bn/virtualization.md" in commit meassag. can you elaborate??

No, you don't have to specify  `[bn] localization update content/bn/virtualization.md` in the commit message, you can submit a Pull Request following the name convention `[bn]  Localize <file_name.md>`, so it will be easy to track Pull Requests.

PrashantDesale2004 commented:
> > > Hey @PrashantDesale2004 Kindly set the targeted branch from `main` to `dev-bn`. You can use `[bn] localization update content/bn/virtualization.md` as PR as I have changed it.
> > 
> > 
> > i didn't get it. Do you mean i have to specify "[bn] localization update content/bn/virtualization.md" in commit meassag. can you elaborate??
> 
> No, you don't have to specify `[bn] localization update content/bn/virtualization.md` in the commit message, you can submit a Pull Request following the name convention `[bn] Localize <file_name.md>`, so it will be easy to track Pull Requests.



> > > Hey @PrashantDesale2004 Kindly set the targeted branch from `main` to `dev-bn`. You can use `[bn] localization update content/bn/virtualization.md` as PR as I have changed it.
> > 
> > 
> > i didn't get it. Do you mean i have to specify "[bn] localization update content/bn/virtualization.md" in commit meassag. can you elaborate??
> 
> No, you don't have to specify `[bn] localization update content/bn/virtualization.md` in the commit message, you can submit a Pull Request following the name convention `[bn] Localize <file_name.md>`, so it will be easy to track Pull Requests.

@sajibAdhi, I have change my commit message, so you can  review it.

sajibAdhi commented on content/bn/virtualization.md:
```diff
@@ -7,25 +7,18 @@ tags: ["fundamental", "infrastructure", "methodology"]
-সেই বিচ্ছিন্ন অপারেটিং সিস্টেম এবং তাদের ডেডিকেটেড কম্পিউট রিসোর্স (সিপিইউ, মেমরি এবং নেটওয়ার্ক)
-ভার্চুয়াল মেশিন বা ভিএম হিসাবে উল্লেখ করা হয়।
-যখন আমরা একটি [ভার্চুয়াল মেশিন](/bn/virtual-machine/) সম্পর্কে কথা বলি, তখন আমরা একটি সফ্টওয়্যার-সংজ্ঞায়িত কম্পিউটারের কথা বলছি।
+ভার্চুয়ালাইজেশন, ক্লাউড নেটিভ কম্পিউটিং সম্পর্কে, <br>-কখনওকি কখনও একটি সার্ভারের সঙ্গে সম্প্রেষণ করে এবং এটি একাধিক বিভিন্ন অপারেটিং সিস্টেম চালাতে দেয়। <br>-এই বিভিন্ন অপারেটিং সিস্টেম এবং তাদের ডেডিকেটেড কম্পিউটিং সম্পদ (সিপিইউ, মেমরি, এবং নেটওয়ার্ক) <br>-ভার্চুয়াল মেশিন বা ভিএম হিসাবে পরিচিত হয়।
```
Hello @PrashantDesale2004.

Kindly remove `<br>` tags. As you can see in the below picture that it is breaking the UI.
![image](https://github.com/cncf/glossary/assets/60180521/794bd662-fa24-4347-8c26-a6f2ac83a19f)
 
 Kindly make those sentence more easy for reader?

sajibAdhi commented:
Hey @PrashantDesale2004., do you have any update on this PR?
If you are working on this PR kindly let me know.

asem-hamid commented:
This issue is already fixed in this [PR](https://github.com/cncf/glossary/pull/2404 )  

@sajibAdhi you can close it now

sajibAdhi commented:
> This issue is already fixed in this [PR](https://github.com/cncf/glossary/pull/2404)
> 
> @sajibAdhi you can close it now

As this issue is already fixed. So I am closing this issue.
